components/Footer/index.tsx

```diff
@@ -1,19 +1,28 @@
-import { SiFacebook as Facebook } from 'react-icons/si'
 import style from './Footer.module.css'
+import utilityStyle from './utilities.module.css'
+import { SiFacebook as Facebook } from 'react-icons/si'
 
 export default function Footer(): JSX.Element {
   return (
-    <footer className={style.footer} id="social">
-      <span>
-        Connect with us on{' '}
-        <a href="https://www.facebook.com/TheJacNZ/" className={style.icon}>
-          <Facebook />
+    <footer className={[style.footer, utilityStyle.bgCloudBurst, utilityStyle.colorLight].join(' ')}>
```

Comment:
What is the advantage to three classes here?

Reply:
The first `style.footer` class defines the component and its children's layout. These styles should be specific for this component. 
The other two classes define the background color and color. They should be consistent with the other part of the website. That says once it was put in a webpage, these two classes should be replaced with the color classes for that specific webpage.

Reply:
Would some of those "utility" classes that are simply colours be better as CSS variables?

Reply:
No. CSS variables are a better way. Thanks!